CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ All notable changes to this project will be documented in this file.
 - Change Webpack to not use @babel/preset-env to compile node_modules ([ykzts](https://github.com/tootsuite/mastodon/pull/10289))
 - Change web UI to use new Web Share Target API ([gol-cha](https://github.com/tootsuite/mastodon/pull/9963))
 - Change ActivityPub reports to have persistent URIs ([ThibG](https://github.com/tootsuite/mastodon/pull/10303))
+- Changed `tootctl accounts cull --dry-run` to show a list of accounts that would be deleted ([BenLubar](https://github.com/tootsuite/mastodon/pull/10460))
```

Comment:
Change `tootctl accounts cull --dry-run` to list of accounts that would be deleted

(Change, not changed)